InternetRadioDevice/InternetRadio/Interfaces/Gpio/GpioInterfaceManager.cs

```diff
@@ -42,25 +33,29 @@ public bool Initialize()
                 return false;
             }
 
-            foreach (var pinSetting in Config.Buttons_Pins)
+            foreach (var pinSetting in buttonPins)
             {
                 GpioPin button;
-                GpioOpenStatus status;
-
-                if (gpio.TryOpenPin(pinSetting.Key, GpioSharingMode.Exclusive, out button, out status))
+                GpioOpenStatus status = GpioOpenStatus.PinUnavailable;
+                try
                 {
-                    if (status == GpioOpenStatus.PinOpened)
+                    if (gpio.TryOpenPin(pinSetting.Key, GpioSharingMode.Exclusive, out button, out status))
                     {
-                        button.DebounceTimeout = new TimeSpan(Config.Buttons_Debounce);
-                        button.SetDriveMode(GpioPinDriveMode.Input);
-                        button.ValueChanged += handleButton;
-                        Debug.WriteLine("Button on pin " + pinSetting.Value + " successfully bound for action: " + pinSetting.Key.ToString());
-                        actionButtons.Add(pinSetting.Value, button);
-                        button = null;
-                        continue;
+                        if (status == GpioOpenStatus.PinOpened)
+                        {
+                            button.DebounceTimeout = new TimeSpan(buttonDebounce);
+                            button.SetDriveMode(GpioPinDriveMode.Input);
+                            button.ValueChanged += handleButton;
+                            Debug.WriteLine("Button on pin " + pinSetting.Value + " successfully bound for action: " + pinSetting.Key.ToString());
+                            actionButtons.Add(pinSetting.Value, button);
+                            button = null;
+                            continue;
```

Comment:
Can we ditch the "continue" here and just put the debug statement in an "else" block?